tensorflow/core/kernels/mkl_concat_op.cc

```diff
@@ -608,7 +657,11 @@ class MklConcatOp : public OpKernel {
             if (input_tensors[k].NumElements() == 0) continue;
             auto src_md = mkl_input_shapes[k].GetMklLayout();
             srcs[k].SetUsrMem(src_md, &input_tensors[k]);
+#ifdef ENABLE_MKLDNN_V1
+            auto src_mpd = srcs[k].GetUsrMemDesc();
+#else
             auto src_mpd = srcs[k].GetUsrMemPrimDesc();
+#endif  // ENABLE_MKLDNN_V1
```

Comment:
Will `GetUsrMemPrimDesc` be used a lot? If so, please define a macro in `mkl_util.h`.
```c++
// In mkl_util.h
#ifdef ENABLE_MKLDNN_V1
#define GET_USR_MEM_PRIM_DESC(src) src.GetUsrMemDesc()
#else 
#define GET_USR_MEM_PRIM_DESC(src) src.GetUsrMemPrimDesc()
#endif  // ENABLE_MKLDNN_V1
...
// In here.
auto src_mpd = GET_USR_MEM_PRIM_DESC(srcs[k]);
```

Reply:
ok